### PR TITLE
Make sure it only calls next action once.

### PR DESCRIPTION
**Pull request by TheMonDon**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [X] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [X] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by almeidx — COMMENTED** [review]
While you're at it, refactor the promises in lines 320 and 368 to use async/await to improve code quality

**Review by almeidx — APPROVED** [review]

File: actions/youtube_search_MOD.js [hunk 1]
```diff
@@ -421,7 +421,7 @@ module.exports = {
                   const storage = parseInt(data.storage)
                   const varName = Actions.evalMessage(data.varName, cache)
                   Actions.storeValue(result2, storage, varName, cache)
-                  Actions.callNextAction(cache)
+                  return Actions.callNextAction(cache)
```
Comment (almeidx) [inline actions/youtube_search_MOD.js:424]: This doesn't change anything since it's inside it's own callback

File: actions/youtube_search_MOD.js [hunk 2]
```diff
@@ -430,7 +430,7 @@ module.exports = {
             const storage = parseInt(data.storage)
             const varName = Actions.evalMessage(data.varName, cache)
             Actions.storeValue(result, storage, varName, cache)
-            Actions.callNextAction(cache)
+            return Actions.callNextAction(cache)
```
Comment (almeidx) [inline actions/youtube_search_MOD.js:433]: This doesn't change anything since it's inside a callback

**Comment by TheMonDon** [comment]
Superseded by #744

**Pull request closed without merging**